dataline-scheduler/src/test/java/io/dataline/scheduler/JobSchedulerTest.java

```diff
@@ -145,64 +100,39 @@ public void setup() {
     when(jobFactory.create(STANDARD_SYNC.getConnectionId())).thenReturn(JOB_ID);
-    setConfigMocks();
 
     scheduler.run();
 
-    verifyConfigCalls();
-    verify(scheduleJobPredicate).test(Optional.of(JobSchedulerTest.PREVIOUS_JOB), STANDARD_SYNC_SCHEDULE);
-    verify(schedulerPersistence).getLastSyncJob(STANDARD_SYNC.getConnectionId());
     verify(jobFactory).create(STANDARD_SYNC.getConnectionId());
   }
 
   @Test
-  public void testScheduleJobNoPreviousJob() throws JsonValidationException, ConfigNotFoundException, IOException {
+  public void testScheduleJobNoPreviousJob() throws IOException {
     when(schedulerPersistence.getLastSyncJob(STANDARD_SYNC.getConnectionId()))
         .thenReturn(java.util.Optional.empty());
-    when(scheduleJobPredicate.test(Optional.empty(), STANDARD_SYNC_SCHEDULE)).thenReturn(true);
+    when(scheduleJobPredicate.test(Optional.empty(), STANDARD_SYNC.getSyncSchedule())).thenReturn(true);
     when(jobFactory.create(STANDARD_SYNC.getConnectionId())).thenReturn(JOB_ID);
-    setConfigMocks();
 
     scheduler.run();
 
-    verifyConfigCalls();
-    verify(scheduleJobPredicate).test(Optional.empty(), STANDARD_SYNC_SCHEDULE);
-    verify(schedulerPersistence).getLastSyncJob(STANDARD_SYNC.getConnectionId());
     verify(jobFactory).create(STANDARD_SYNC.getConnectionId());
   }
 
   @Test
-  public void testDoNotScheduleJob() throws JsonValidationException, ConfigNotFoundException, IOException {
+  public void testDoNotScheduleJob() throws IOException {
     when(schedulerPersistence.getLastSyncJob(STANDARD_SYNC.getConnectionId()))
         .thenReturn(java.util.Optional.of(PREVIOUS_JOB));
-    when(scheduleJobPredicate.test(Optional.of(JobSchedulerTest.PREVIOUS_JOB), STANDARD_SYNC_SCHEDULE)).thenReturn(false);
-    setConfigMocks();
+    when(scheduleJobPredicate.test(Optional.of(JobSchedulerTest.PREVIOUS_JOB), STANDARD_SYNC.getSyncSchedule()))
+        .thenReturn(false);
 
     scheduler.run();
 
-    verifyConfigCalls();
-    verify(scheduleJobPredicate).test(Optional.of(JobSchedulerTest.PREVIOUS_JOB), STANDARD_SYNC_SCHEDULE);
-    verify(schedulerPersistence).getLastSyncJob(STANDARD_SYNC.getConnectionId());
     verify(jobFactory, never()).create(STANDARD_SYNC.getConnectionId());
   }
 
-  // sets all mocks that are related to fetching configs. these are the same for all tests in this
-  // test suite.
-  private void setConfigMocks() throws JsonValidationException, ConfigNotFoundException, IOException {
-    when(configRepository.listStandardSyncs()).thenReturn(Collections.singletonList(STANDARD_SYNC));
-    when(configRepository.getStandardSyncSchedule(STANDARD_SYNC.getConnectionId())).thenReturn(STANDARD_SYNC_SCHEDULE);
-  }
-
-  // verify all mocks that are related to fetching configs are called. these are the same for all
-  // tests in this test suite.
-  private void verifyConfigCalls() throws ConfigNotFoundException, IOException, JsonValidationException {
```

Comment:
These verifies aren't necessary. they are already "tested" by the fact that they are mocked. if these calls weren't made the job factory wouldn't be called.